Formula/latexdiff.rb

```diff
@@ -0,0 +1,43 @@
+class Latexdiff < Formula
+  desc "Determine and markup differences between two LaTeX files"
+  homepage "https://www.ctan.org/pkg/latexdiff"
+  url "https://github.com/ftilmann/latexdiff/releases/download/1.2.1/latexdiff-1.2.1.tar.gz"
+  sha256 "12634c8ec5c68b173d3906679bb09330e724491c6a64e675989217cf4790604e"
+
+  def install
+    # Replace standard latexdiff with latexdiff-so which uses an inlined
+    # version of the Perl algorithm Algorithm::Diff.
+    # This is the preferred method according to the README
+    cp "latexdiff-so", "latexdiff"
+    bin.install %w[latexdiff latexdiff-fast latexdiff-so latexdiff-vc
```

Comment:
Let's install `latexdiff-so`, do not install `latexdiff`, and then create a symlink of `latexdiff-so` as `latexdiff`

Reply:
Changed as requested